### ISSUE TITLE
There was an overfitting phenomenon in the visual front and back

### ISSUE THREAD

**Issue by qq2418916003** [LUMIA-Group/Leveraging-Self-Supervised-Learning-for-AVSR#9]
When training the vo model, it seems that there is overfitting linearity. When the wer on the training set is 0.77+, the wer on the training set is about 0.47, and when the wer on the training set is about 0.74+, the wer on the training set is still about 0.47. 
Do I miss the best model on the test set？
